Sources/Dependencies/Dependencies/OpenURL.swift

```diff
@@ -22,7 +22,7 @@ private enum OpenURLKey: DependencyKey {
   static let liveValue = OpenURLEffect { url in
     let stream = AsyncStream<Bool> { continuation in
       let task = Task { @MainActor in
-        #if canImport(AppKit)
+        #if canImport(AppKit) && !targetEnvironment(macCatalyst)
```

Comment:
Alternatively, we can be more explicit about the intent when selecting platforms and simplify the logic by removing negations using `os`:
```swift
#if os(iOS) && targetEnvironment(macCatalyst)

#elseif os(iOS)

#elseif os(tvOS)

#elseif os(watchOS)

#elseif os(macOS)

#else 

#endif
```

Reply:
Might revisit this in the future, as it is admittedly a bit confusing, but for now just focused on fixing the build 😅